### ISSUE TITLE
NPE in CacheDetailActivity

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#5063]
From Google Play for versions 2015.06.15 and 2015.06.19 on Android 4.4 and 5.x devices:

```
java.lang.IllegalStateException: Exception thrown on Scheduler.Worker thread. Add `onError` handling.
  at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:60)
  at android.os.Handler.handleCallback(Handler.java:743)
  at android.os.Handler.dispatchMessage(Handler.java:99)
  at android.os.Looper.loop(Looper.java:136)
  at android.app.ActivityThread.main(ActivityThread.java:5212)
  at java.lang.reflect.Method.invokeNative(Native Method)
  at java.lang.reflect.Method.invoke(Method.java:515)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:602)
  at dalvik.system.NativeStart.main(Native Method)
Caused by: rx.exceptions.OnErrorNotImplementedException
  at rx.Observable$30.onError(Observable.java:7358)
  at rx.observers.SafeSubscriber._onError(SafeSubscriber.java:154)
  at rx.observers.SafeSubscriber.onError(SafeSubscriber.java:111)
  at rx.observers.SafeSubscriber.onNext(SafeSubscriber.java:137)
  at rx.internal.operators.OperatorSubscribeOn$1$1$1.onNext(OperatorSubscribeOn.java:76)
  at rx.android.app.OperatorConditionalBinding$1.onNext(OperatorConditionalBinding.java:79)
  at rx.internal.operators.OperatorObserveOn$ObserveOnSubscriber.pollQueue(OperatorObserveOn.java:208)
  at rx.internal.operators.OperatorObserveOn$ObserveOnSubscriber$2.call(OperatorObserveOn.java:170)
  at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
  ... 9 more
Caused by: java.lang.NullPointerException
  at butterknife.ButterKnife.findById(ButterKnife.java:380)
  at cgeo.geocaching.CacheDetailActivity$DetailsViewCreator$1.call(CacheDetailActivity.java:907)
  at cgeo.geocaching.CacheDetailActivity$DetailsViewCreator$1.call(CacheDetailActivity.java:902)
  at rx.Observable$30.onNext(Observable.java:7363)
  at rx.observers.SafeSubscriber.onNext(SafeSubscriber.java:130)
... 14 more
```

```
java.lang.NullPointerException
  at butterknife.ButterKnife.findById(ButterKnife.java:380)
  at cgeo.geocaching.CacheDetailActivity$DetailsViewCreator.toggleAttributesView(CacheDetailActivity.java:1036)
  at cgeo.geocaching.CacheDetailActivity$DetailsViewCreator$2.onItemClick(CacheDetailActivity.java:1030)
  at android.widget.AdapterView.performItemClick(AdapterView.java:308)
  at android.widget.AbsListView.performItemClick(AbsListView.java:1476)
  at android.widget.AbsListView$PerformClick.run(AbsListView.java:3497)
  at android.widget.AbsListView$3.run(AbsListView.java:4814)
  at android.widget.AbsListView.onDetachedFromWindow(AbsListView.java:3316)
  at android.view.View.dispatchDetachedFromWindow(View.java:13462)
  at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:2791)
  at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:2789)
  at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:2789)
  at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:2789)
  at android.view.ViewGroup.removeViewInternal(ViewGroup.java:4056)
  at android.view.ViewGroup.removeViewInternal(ViewGroup.java:4029)
  at android.view.ViewGroup.removeView(ViewGroup.java:3961)
  at android.support.v4.view.ViewPager.removeView(ViewPager.java:1326)
  at cgeo.geocaching.activity.AbstractViewPagerActivity$ViewPagerAdapter.destroyItem(AbstractViewPagerActivity.java:130)
  at android.support.v4.view.ViewPager.dataSetChanged(ViewPager.java:874)
  at android.support.v4.view.ViewPager$PagerObserver.onChanged(ViewPager.java:2824)
  at android.database.DataSetObservable.notifyChanged(DataSetObservable.java:37)
  at android.support.v4.view.PagerAdapter.notifyDataSetChanged(PagerAdapter.java:276)
  at cgeo.geocaching.activity.AbstractViewPagerActivity.reinitializeViewPager(AbstractViewPagerActivity.java:289)
  at cgeo.geocaching.CacheDetailActivity.notifyDataSetChanged(CacheDetailActivity.java:744)
  at cgeo.geocaching.CacheDetailActivity.onLoadFinished(CacheDetailActivity.java:215)
  at cgeo.geocaching.CacheDetailActivity.onLoadFinished(CacheDetailActivity.java:153)
  at android.support.v4.app.LoaderManagerImpl$LoaderInfo.callOnLoadFinished(LoaderManager.java:427)
  at android.support.v4.app.LoaderManagerImpl$LoaderInfo.onLoadComplete(LoaderManager.java:395)
  at android.support.v4.content.Loader.deliverResult(Loader.java:104)
  at android.support.v4.content.AsyncTaskLoader.dispatchOnLoadComplete(AsyncTaskLoader.java:223)
  at android.support.v4.content.AsyncTaskLoader$LoadTask.onPostExecute(AsyncTaskLoader.java:61)
  at android.support.v4.content.ModernAsyncTask.finish(ModernAsyncTask.java:461)
  at android.support.v4.content.ModernAsyncTask.access$500(ModernAsyncTask.java:47)
  at android.support.v4.content.ModernAsyncTask$InternalHandler.handleMessage(ModernAsyncTask.java:474)
  at android.os.Handler.dispatchMessage(Handler.java:102)
  at android.os.Looper.loop(Looper.java:157)
  at android.app.ActivityThread.main(ActivityThread.java:5335)
  at java.lang.reflect.Method.invokeNative(Native Method)
  at java.lang.reflect.Method.invoke(Method.java:515)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1265)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1081)
  at dalvik.system.NativeStart.main(Native Method)
```

**Comment by rsudev** [comment]
Could be related to #4978 
Some view, which normally is there is already destroyed it seems, but I couldn't find a clear scenario to reproduce this.

**Comment by Lineflyer** [comment]
No more crash reports received which match to this in the last published releases.
Therefore closing. Might be fixed together with the fix for #4978
